setup.py

```diff
@@ -24,7 +24,7 @@
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
     python_requires=">=3.8",
-    install_requires=["tutor <16, >=14.0.0"],
+    install_requires=["tutor>=14.0.0"],
```

Comment:
Please don't just drop the upper version bound, but instead replace `<16` with `<17`.